tool/generate_code.sh

```diff
@@ -10,37 +10,9 @@ RELATIVE_PATH_TO_SCRIPT="${BASH_SOURCE[0]}"
 # The directory that this script is located in.
 TOOL_DIR=`dirname "${RELATIVE_PATH_TO_SCRIPT}"`
```

Comment:
lets just delete this file altogether and call `devtools_tool generate-code` everywhere we were previously calling generate_code.sh

![Screenshot 2023-10-24 at 9 25 51 AM](https://github.com/flutter/devtools/assets/43759233/299e52c2-feef-4f29-bed8-56849f409f69)

Reply:
Done! I wasn't sure if others might have scripts/keybinds to call it, but this probably makes more sense. I'll ping a note in Discord about it too.